data/com.github.cjbassi.ytop.appdata.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- 2020 Artem Polishchuk <ego.cordatus@gmail.com> -->
+<component type="console-application">
+  <id>com.github.cjbassi.ytop</id>
+  
+  <name>ytop</name>
+  <summary>A TUI system monitor written in Rust</summary>
+  
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MIT</project_license>
+  
+  <description>
+    <p>
+      Another TUI based system monitor, this time in Rust!
+    </p>
+  </description>
+  
+  <categories>
```

Comment:
Should be defined in the desktop file. They are merged with appstream-compose later.